studies/tasks.py

```diff
@@ -526,7 +526,7 @@ def get_all_incomplete_video_files():
     except ParamValidationError as error:
         raise ValueError(f"The parameters you provided are incorrect: {error}")
 
-    if uploads_response["Uploads"]:
+    if "Uploads" in uploads_response:
         # Filter out incomplete uploads that might still be actively recording - started in last 24 hours.
```

Comment:
This assumes that `uploads_response["Uploads"]' is always assigned.  Are we certain it will be?

Reply:
I think this assumes that `uploads_response` exists and tests if "Uploads" is a key that exists in the dictionary. 
But this is a great point - I don't think I've done enough here to prevent other/similar errors. I've updated this PR with a few more changes to this logic and tests.

Reply:
Oooh, I see, do you mean that I need to check for something like `uploads_response["Uploads"]` exists but is null/None?

Reply:
After the API call, I would check that `uploads_response` is not None.  Then I would check if there's a key for 'Uploads' and if the value at that key is not "None": `"Uploads" in uploads_response and uploads_response["Uploads"].`

Reply:
It's basically what you had before combined with what you have now.

Reply:
Oh sorry I forgot to point out in my comment that I'd also updated this code to
```
if uploads_response is not None and "Uploads" in uploads_response:
```

Reply:
@okaycj Can you have a look at this one more time? I moved the bit that handles the multipart uploads response into try/except, and did the same with the list parts response handling (same problem there - the "Parts" key is missing from the response when there are no parts). 

I also added some extra validation logic that I missed before, and more tests.

And I did test this on our S3 buckets and verified that it works. (At least, it handles the cases that I was able to test with our buckets, which are (1) incomplete uploads that can be completed, (2) uploads that were created but could not be completed because they do not have any associated parts).